src/config/entities/__tests__/configuration.ts

```diff
@@ -38,6 +38,14 @@ export default (): ReturnType<typeof configuration> => ({
           56: { chainName: faker.string.sample() },
           8453: { chainName: faker.string.sample() },
         },
+        currencies: [
+          ...Array.from({ length: faker.number.int({ min: 2, max: 5 }) }, () =>
+            faker.finance.currencyCode(),
+          ),
+          'ETH',
+          'EUR',
+          'USD',
+        ],
```

Comment:
Do we need the harcoded values here? There could potentially be duplicates.

Reply:
> Do we need the harcoded values here?

I hardcoded some currency codes in the test configuration to be able to make some assertions at `balances.controlller.spec.ts`. Otherwise, we can't figure out the result of the intersection between providers' configured codes because we are not mocking the configuration at the Controller tests. I think doing so would add noise to the PR. But ofc, let me know if you think there's a better way to test the intersection without hardcoding some codes in the test configuration 🙂

> There could potentially be duplicates

I agree! It may be more correct to avoid duplicates (even if the intersection would hide the duplication). I've fixed it in [d6f35a0](https://github.com/safe-global/safe-client-gateway/pull/1099/commits/d6f35a006240b85a4e1f2a9743a33342c0b1e1c4)